docs/home/whats-new-2-18.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Version 2.18
+summary: This article describes new features and functionality in the version.
+authors:
+    - Jamie Weider
+date: 2024-June-14
+---
+
+## Release Content - June 30, 2024
+
+* [Deprecation notifications](#deprecation-notifications)
+* [Breaking changes](#breaking-changes)
+
+### Researcher
+
+#### Jobs, Workloads, and Workspaces
+
```

Comment:
Add context that this capability is already supported in the API and CLI. A link to these references would be helpful